nuttx/include.am

```diff
@@ -0,0 +1,10 @@
+EXTRA_DIST+= nuttx/include.am
```

Comment:
You cannot include `nuttx/include.am`. Please add to `Makefile.am`.

Include the standard include.am header:

```
# vim:ft=automake
# included from Top Level Makefile.am
# All paths should be given relative to the root
```

The whole point of this is to include it with `make dist`.